releng/org.eclipse.nebula.feature/feature.xml

```diff
@@ -261,4 +261,8 @@
          id="org.eclipse.nebula.widgets.treemapper.feature"
          version="0.0.0"/>
 
+   <includes
```

Comment:
If it's added here, I would expect it be removed from somewhere else, i.e., from org.eclipse.nebula.incubation.feature.

Reply:
ack, fixed in commit 2